sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonSuite.scala

```diff
@@ -2563,4 +2563,18 @@ class JsonSuite extends QueryTest with SharedSQLContext with TestJsonData {
       assert(!files.exists(_.getName.endsWith("json")))
     }
   }
+
+  test("return partial result for bad records") {
+    val schema = new StructType()
+      .add("a", DoubleType)
+      .add("b", ArrayType(IntegerType))
+      .add("c", StringType)
+      .add("_corrupt_record", StringType)
```

Comment:
Shall we use the following simple one-liner?
```scala
val schema = "a double, b array<int>, c string, _corrupt_record string"
```

Reply:
replaced